ranger/data/scope.sh

```diff
@@ -50,6 +50,30 @@ OPENSCAD_COLORSCHEME="${RNGR_OPENSCAD_COLORSCHEME:-Tomorrow Night}"
 SQLITE_TABLE_LIMIT=20  # Display only the top <limit> tables in database, set to 0 for no exhaustive preview (only the sqlite_master table is displayed).
 SQLITE_ROW_LIMIT=5     # Display only the first and the last (<limit> - 1) records in each table, set to 0 for no limits.
 
+# HELPERS
+# =============================================================================
+pygmentize() {
```

Comment:
Okay this part is just the lines moved from `text/* | */xml`, former line 417.

Is it okay to name a function the same as `pygmentize` utility?

Reply:
It implements the utility pygmentize. That's why the name is pygmentize.

Yes it is the same code. We just made it a function so it can be re-used in the code.